projects/batfish-client/src/org/batfish/client/Client.java

```diff
@@ -1393,7 +1578,7 @@ private boolean processCommand(String[] words, FileWriter outWriter) {
          case LIST_TESTRIGS:
             return listTestrigs();
          case LOAD_QUESTIONS:
-            return loadQuestions(outWriter, parameters);
+            return loadQuestions(outWriter, parameters, _bfq);
```

Comment:
To be able to load dictionaries in a supplied dictionary variable, instead of always using the class member. I am using it to initialize analysis or modify analysis from a directory of questions.